src/Schema/Event.ts

```diff
@@ -93,3 +105,203 @@ export interface SuccessfullyLoggedIn {
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  horizontal_slide_position?: number
+  module_height?: "single" | "double"
+  type?: "thumbnail" | "header" | "stub"
+}
+
+/**
+ * A user taps a grouping of auctions on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedAuctionGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedAuctionGroup"
+ *    context_module: "auctionsRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Sale
+ *    destination_screen_owner_id: "5e95b37a2fdcb20012a0e082"
+ *    destination_screen_owner_slug: "forum-auctions-colour-theory-4"
+ *    horizontal_slide_position: 3
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedAuctionGroup {
+  action: ActionType.tappedAuctionGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  horizontal_slide_position?: number
+  module_height?: "single" | "double"
+  type?: "thumbnail" | "header" | "stub"
+}
+
+/**
+ * A user taps a grouping of collections on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedCollectionGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedCollectionGroup"
+ *    context_module: "collectionRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Collection
+ *    destination_screen_owner_slug: "limited-edition-prints-trending-artists"
+ *    horizontal_slide_position: 2
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedCollectionGroup {
+  action: ActionType.tappedCollectionGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  horizontal_slide_position?: number
+  module_height?: "single" | "double"
+  type: "thumbnail" | "header" | "stub"
+}
+
+/**
+ * A user taps a grouped explore module (City Guide, for example)
+ *
+ * This schema describes events sent to Segment from [[tappedExploreGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedExploreGroup"
+ *    context_module: "cityGuideCard"
+ *    context_screen_owner_type: Search
+ *    destination_screen_owner_type: CityGuideGuide
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedExploreGroup {
```

Comment:
right now, this is intended to track the new city guide entry path, but I was imagining we might have other 'explore' experiences in the future, so I left this event as open and generic as possible. 

One thing I was wondering -- right now we call the screen this card sits on the `Search` screen, but the new designs have the bottom nav referring to the page icon as `Explore`. Should we standardize here? I think `Search` is good for the screen type because it's an existing screen/page type, even if the copy/UI says otherwise.

Reply:
Ooh good call, actually we might want to refer to this page (bottom of stack) as `Explore` and the page when you tap into the search input bar `Search` (new page on top).

We'll eventually move the recent searches to the `Search` screen, plus potentially like trending searches, etc.
![image](https://user-images.githubusercontent.com/6038013/80246945-43636000-863b-11ea-94f4-85f7914cecf8.png)

But we'll keep `explore` for just general browsing:
![image](https://user-images.githubusercontent.com/6038013/80246921-38103480-863b-11ea-84d5-c9a11a538b57.png)